core/base/data-updaters/document-data-updater.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Elementor\Core\Base\Data_Updaters;
+
+use Elementor\Element_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+abstract class Document_Data_Updater {
+	protected $document;
+
+	abstract public function is_update_needed();
+
+	public function update_unique_widget( Element_Base $element_data ) {}
```

Comment:
Rename to `unique_element_iteration`

Reply:
Done, @kobizz - I've also changed: `is_update_needed` to: `is_action_needed`.